### ISSUE TITLE
--depth STANDARD doesn't skip DEEP scans

### ISSUE THREAD

**Issue by mcouillard** [SonarSource/sonarqube-cli#769]
### CLI version

1.7.0

### Operating system and version

Windows 11

### Command you ran

```shell
sonar analyze --depth STANDARD
```

### What happened

```shell
...
  ✓  File1.js

✅ No issues found · 1289ms
...
  ✗  File1.js
     SonarQube API error: 404 Not Found - {"message":"Resource not found."}
     → Check your SonarQube Cloud authentication, project key, and network connectivity, then retry.

30 files analyzed · 30 failures · STANDARD analysis
```

### What you expected to happen

No 404 errors related to DEEP scanning.

### Additional context

Sonarqube Server is 2026.4.1 Enterprise

**Comment by mcouillard** [comment]
I've since tried older CLI versions, then back to 1.7.0, and updated the Sonar MCP server version. I'm no longer getting this problem.
